### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by ignatius22**
I have completed the following task for the milestone 3 assignment.
 - Used flex display mode for App, Display and ButtonPanel components.
- Styled the Display component.
- Styled the ButtonPanel component.
- Styled the Button component.
- Pass "color" and "wide" props to the Button

Please take a moment to review my project. I'm looking forward to your feedback. Thanks.

### CODE REVIEW

File: src/components/ButtonPanel/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from '../Button/Button';
+import './ButtonPanel.css';
+
+const grey = {
+  color: '#dfdfdf',
+};
+
+const ButtonPanel = () => (
+  <>
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel/ButtonPanel.js]: **OPTIONAL**: You might not need this Fragment, you are already wraping everything in a `div`.

File: src/components/ButtonPanel/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from '../Button/Button';
+import './ButtonPanel.css';
+
+const grey = {
+  color: '#dfdfdf',
+};
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel/ButtonPanel.js]: 🤔 shouldn't the color being passed to the `Button` component be a string?

File: src/components/ButtonPanel/ButtonPanel.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <Button name="1" color={grey} />
+        <Button name="2" color={grey} />
+        <Button name="3" color={grey} />
+        <Button name="+" />
+      </div>
+      <div className="group">
+        <Button name="0" color={grey} wide />
+        <Button name="." color={grey} />
+        <Button name="=" />
+      </div>
```
Comment (MauricioRobayo) [inline src/components/ButtonPanel/ButtonPanel.js]: **OPTIONAL**: So much duplicate code... could we somehow avoid all this repeated code? 🤨

<details>
 <summary>Hint</summary>
 
You could specify the data that you need on an array of arrays or an object holding arrays for each group, and map through those.

</details>

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
This is looking great, well done so far, nice looking code, well organized and readable.

Before continuing, please double-check the color prop of the Button component. There is a warning on the browser's console:

```
ButtonPanel.js:11 Warning: Failed prop type: Invalid prop `color` of type `object` supplied to `Button`, expected `string`.
```

- [x] Fix color prop for Button component.

Other than that, the project looks great!

Status: **Request changes**.

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great job implementing the suggested changes.

**You may proceed to the next milestone.**

**Happy Coding :computer: 
Simon Wathigo.**

**Pull request merged**